app/controllers/api/board_controller.rb

```diff
@@ -9,5 +9,11 @@ def commit
       render json: huboard.board(params[:user], params[:repo]).commit(params[:commit])
     end
 
+    def issue_template
+      board = huboard.board(params[:user], params[:repo])
+      template = board.issue_template
+      decoded_template = template ? Base64.decode64(template['content']) : nil
```

Comment:
I'm surprised to have a `Base64.decode64` in two places. Could the decode move down into bridge?